### ISSUE TITLE
gmail not forwarding emails with backed up sms

### ISSUE THREAD

**Issue by lvminers** [jberkel/sms-backup-plus#919]
Hi! 
My task is to set up forwarding of emails with backed up SMS to another email address. 
Gmail doen't do that with backed up SMS, however all other emails are forwarded as expected.
I process this either via filter set up (sorted by "SMS" in subject and forward option) or forward ALL incoming emails - no difference.

I tried to change the default SMS tag to Inbox so that all backed up SMS so up in Inbox folder - same result.

The email sender of the backed up SMS looks strange: (sender name)@unknown.email 
Could this @unknown.email  be the reason why gmail can't handle it?

Best regards,
Leonid

**Comment by basti-exp360** [comment]
**tl;dr**: Please **allow the label setting to be blank** in the app settings.
This would then skip adding a label for the backed up SMS messages. Why? See below..

Hi, I've encountered the same issue yesterday and spent lots of time trying to find a solution.
My feeling is that there are two likely reasons for this:

1. Since the SMS are backed up into the gmail accounts, they do not get 'received' and are therefore not processed by the filters correctly (which would be hard to fix).
2. Maybe they are also not processed just because they get labeled by SMSbackup+ (which is what I hope for). Reason for this thinking: when you search for a label and then continue to create a filter for this search, there is a warning that this filter will not work due to the label being used as a filter.
However, at this point you cannot leave the label setting blank in the app. Would be good to see what happens when you didn't have to put in a label...

Cheers,
Bastian
